src/commcare_cloud/ansible/roles/backups/templates/check_s3_backup.py.j2

```diff
@@ -34,7 +35,13 @@ s3 = session.resource(service_name='s3',endpoint_url=endpoint,region_name='{{ aw
 s3 = session.resource(service_name='s3',endpoint_url=endpoint)
 {% endif %}
 
-bucket = s3.Bucket(BUCKET_NAME)
+def file_exists_in_s3(object_key):
+  try:
+    s3.Object(BUCKET_NAME, object_key).load()
+  except botocore.exceptions.ClientError:
```

Comment:
nit: not sure how much it matters here, but `ClientError` does not necessarily mean "not found." Here's [how the blob db detects that condition
](https://github.com/dimagi/commcare-hq/blob/03499156b26de39121531b82e7ddbd5aa5405783/corehq/blobs/s3db.py#L184-L186) (still kinda messy)

```py
def is_not_found(err, not_found_codes=["NoSuchKey", "NoSuchBucket", "404"]):
    return (err.response["Error"]["Code"] in not_found_codes or
        err.response.get("Errors", {}).get("Error", {}).get("Code") in not_found_codes)
```

Reply:
I think in our case I'm okay with the mild sloppiness here. If we end up catching errors here that aren't because the file is missing, the worst that'll happen is that we get a false positive here. I might regret this later but I think for now I'll just leave this part as-is